test/multiverse/suites/active_record_pg/active_record_test.rb

```diff
@@ -591,8 +591,8 @@ def adapter
     # ActiveRecord::Base.configurations[NewRelic::Control.instance.env]['adapter']
     #       ActiveRecord::Base.configs_for(env_name: NewRelic::Control.instance.env)['adapter']
     #       ActiveRecord::Base.configs_for(env_name: RAILS_ENV)['adapter']
-    # require 'pry'; binding.pry
     # Order.configurations[RAILS_ENV]['adapter']
+    
```

Comment:
```suggestion

```

It looks like Rubocop had an issue with the whitespace on this LOC. After this change, we should be good to go!

Reply:
Resolved